File: CS50/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = get_int("Height: ");
        if (n >= 1) //|| n >= 8)
        {
            for (int i = 0; i < n; i++)
            {
                for (int j = 0; j < n; j++)
                {
                    printf("#");
                }
            }
        }   printf("\n");
        return 0;
 }

File: CS50/sentimental-mario-less/mario.py
from cs50 import get_int


def main():
    height = get_height()
    for i in range(height):
        for j in reversed(range(height)):
            if j > i:
                print(" ", end="")
        for k in range(height):
            if k <= i:
                print("#", end="")
        print()


def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            return n


main()
